src/runtime_src/core/pcie/driver/linux/xocl/subdev/icap.c

```diff
@@ -2756,9 +2840,11 @@ static int icap_parse_bitstream_axlf_section(struct platform_device *pdev,
 			vfree(*target);
 			*target = NULL;
 		}
-		ICAP_INFO(icap, "skip kind %d, return code %ld", kind, err);
+		ICAP_INFO(icap, "skip kind %d(%s), return code %ld", kind,
+			xrt_xclbin_kind_to_string(kind), err);
 	} else
-		ICAP_INFO(icap, "found kind %d", kind);
+		ICAP_INFO(icap, "found kind %d(%s)", kind,
+			xrt_xclbin_kind_to_string(kind));
```

Comment:
I remember if we have two rows, we'd better add a pair of bracket

Reply:
fixed thanks